cirq-core/cirq/ops/gateset.py

```diff
@@ -76,10 +91,16 @@ def __init__(
             description: Human readable description of the gate family.
             ignore_global_phase: If True, value equality is checked via
                 `cirq.equal_up_to_global_phase`.
+            accepted_tags: If set, only TaggedOperations containing at least one tag in this
+                sequence can be accepted.
+            rejected_tags: If set, a TaggedOperation is rejected if it contains at least one tag in
+                this sequence. This cannot be set if `accepted_tags` is set because tags which are
+                not in `accepted_tags` are already rejected.
```

Comment:
Ah good point, and the other option of prioritizing `accepted_tags` over `rejected_tags` doesn't make sense since the user could then just leave `rejected_tags` empty. Will update.